airflow/api/common/experimental/get_task.py

```diff
@@ -38,3 +38,19 @@ def get_task(dag_id, task_id):
 
     # Return the task.
     return dag.get_task(task_id)
+
+
+def get_task_as_dict(dag_id, task_id):
+    """Return the task object as a dictionary identified by the given dag_id and task_id"""
+    try:
+        task = get_task(dag_id, task_id)
+        fields = {k: str(v)
+                  for k, v in vars(task).items()
+                  if not k.startswith('_')}
+        fields.update({
+            'upstream_task_ids': list(task._upstream_task_ids),
+            'downstream_task_ids': list(task._downstream_task_ids)})
+    except AirflowException as err:
+        raise err
```

Comment:
```suggestion
        raise
```

will preserve the stack traces better

Reply:
Just remove the `try`/`except` entirely and let any error be thrown.

Reply:
done.